src/wizards/generateProject/generationWizard.ts

```diff
@@ -58,14 +58,35 @@ export async function generateProjectWizard() {
     });
 
     state.buildTool = (await input.showQuickPick({
-      title: 'Quarkus Tools',
+      title: INPUT_TITLE,
       step: input.getStepNumber(),
       totalSteps: state.totalSteps,
       placeholder: 'Pick build tool',
       items: quickPickItems,
       activeItem: quickPickItems[0]
     })).label;
 
+    return (input: MultiStepInput) => inputPlatformVersion(input, state);
+  }
+
+  async function inputPlatformVersion(input: MultiStepInput, state: Partial<ProjectGenState>) {
+
+    const quickPickItems: PlatformVersionPickItem[] = await getCodeQuarkusApiPlatforms();
+
+    // Place the recommended option first in the quick pick list
+    quickPickItems.sort((x: PlatformVersionPickItem, y: PlatformVersionPickItem) => {
+      return (x.recommended === y.recommended) ? 0 : x.recommended ? -1 : 1;
```

Comment:
Should we sort by `quarkusCoreVersion` when both have the same recommendation value ? Not sure if the `/streams` response maintains any ordering.

Reply:
Would it be possible for multiple platforms to be recommended?

Reply:
I think it could happen if there were many choices for versions, but for a small amount it would be a bit strange. With that said, we should still probably assume it may happen.